helm/cluster-vsphere/templates/_helpers.tpl

```diff
@@ -96,6 +96,65 @@ See https://github.com/kubernetes-sigs/cluster-api/pull/5027/files
 */}}
 {{- define "kubeadmConfigTemplateSpec" -}}
 {{- include "sshUsers" . }}
+format: ignition
```

Comment:
these are exactly same 58 lines as those in `kubeadmcontrolplane.yaml` (up to the indentation, leading 4 spaces). If it is yet another template defined in this file, it could have been included here (w/ no leading spaces) and also in `kubeadmcontrolplane.yaml` context with something like `{{- include "fooBarIgnition" $ | nindent 8 }}`

wdyt?

Reply:
I pushed the PR without polishing it to not lose my changes. I just tested again with our own Flatcar image and updated. I applied this change too.